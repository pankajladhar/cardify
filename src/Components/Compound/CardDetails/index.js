import React, { Component } from 'react';

class CardDetails extends Component {
    render() {
        return(
            <div className="CardDetails">
                Card Details Component { this.props.match.params.cardId  }
            </div>
        )
    }
}

export default CardDetails;