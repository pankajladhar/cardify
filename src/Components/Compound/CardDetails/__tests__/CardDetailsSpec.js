import React from 'react';
import renderer from 'react-test-renderer';
import CardDetails from './../index';

describe("CardDetails", () => {
    let cardDetailsProps = {}

    beforeEach(() => {
        cardDetailsProps = {
            match: {
                params: {
                    cardId: "sdsd4523sw"
                }
            }
        }
    });

    it('should render correctly', () =>{
        let tree = renderer.create(
            <CardDetails {...cardDetailsProps}/>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
})