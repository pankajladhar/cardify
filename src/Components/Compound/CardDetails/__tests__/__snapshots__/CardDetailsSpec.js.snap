// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardDetails should render correctly 1`] = `
<div
  className="CardDetails"
>
  Card Details Component 
  sdsd4523sw
</div>
`;
